### PR TITLE
chore: Fix build of open-lens

### LINKED ISSUE #7689: `build:app` for open-lens has been failing on macOS
This is causing integration tests to fail for PRs

### PR DESCRIPTION
**Pull request by Nokel81**
- Remove need for using relative paths during build

Signed-off-by: Sebastian Malton <sebastian@malton.name>

This PR will fix the integration test failure on macOS.

### CODE REVIEW

File: open-lens/webpack/renderer.ts [hunk 1]
```diff
@@ -117,79 +118,83 @@
...
+  return [
+    {
+      test: /\.svg$/,
+      type: "asset/source", // exports the source code of the asset, so we get XML
+    },
+    {
+      test: /\.(jpg|png|ico)$/,
+      type: "asset/resource",
+      generator: {
+        filename: "images/[name][ext]",
```
Comment (jim-docker) [inline open-lens/webpack/renderer.ts:148]:
```suggestion
        filename: "icons/[name][ext]",
```
? Or just keeping the svg separate? I don't really follow how they are accessed

File: packages/generate-tray-icons/src/index.ts [hunk 2]
```diff
@@ -41,11 +42,21 @@ const joinWithInitCwd = (relativePath: string): string => {
...
-const spinnerFile = joinWithInitCwd(assertOption("--spinner-icon"));
+const inputFile = await resolve(assertOption("--input"));
+const noticeFile = await resolve(assertOption("--notice-icon"));
+const spinnerFile = await resolve(assertOption("--spinner-icon"));
+
+assert(inputFile, "input file not found");
+assert(noticeFile, "notice icon file not found");
+assert(spinnerFile, "spinner icon file not found");
+
+console.log(inputFile, noticeFile, spinnerFile)
```
Comment (jim-docker) [inline packages/generate-tray-icons/src/index.ts]: is this just for debug?

**Review by jim-docker — DISMISSED** [review]

File: packages/generate-tray-icons/src/index.ts (hunk 2 above)
Comment (Nokel81) [reply packages/generate-tray-icons/src/index.ts]: yes should remove

File: open-lens/webpack/renderer.ts (hunk 1 above)
Comment (Nokel81) [reply open-lens/webpack/renderer.ts:148]: They are separate and of different assert style. see line 142 above. They are `assert/source` which means that the `import X from "<path-to-svg>"` get converted to `const X = "<some-string>";`

**Review by gabriel-mirantis — DISMISSED** [review]

**Review by gabriel-mirantis — APPROVED** [review]

**Pull request merged**